packages/server/validators/events.js

```diff
@@ -37,60 +35,62 @@ const createEventValidator = [
   body("description", "Invalid description")
     .trim()
     .escape()
+    .optional()
     .isLength({ max: 2000 })
     .withMessage("Description max length is 2,000 characters"),
 
   //Location checks
   body("location", "Invalid location")
     .trim()
     .escape()
+    .optional()
     .isLength({ max: 255 })
     .withMessage("Location max length is 255 characters"),
 
   //Thumbnail checks
   body("thumbnail", "Invalid thumbnail")
     .trim()
     .escape()
+    .optional()
     .isLength({ max: 255 })
     .withMessage("Thumbnail max length is 255 characters")
     .custom(async (value) => {
       if (!/\.(jpg|png|JPG|PNG|JPEG|jpeg)$/.test(value)) {
         throw new Error("Thumbnail is not a valid image file");
       }
```

Comment:
You don't need to use a custom validator for regex, express-validator comes with one for regex built in called `.matches()`.
```js
.matches(/regex here/),
```